app/views/hub/notes/index.html.erb

```diff
@@ -7,30 +7,28 @@
 
   <%= render "hub/clients/navigation" %>
 
-  <% if @all_notes_by_day.present? %>
-    <ul class="message-list">
-      <% @all_notes_by_day.each_with_index do |(datetime, records), day_index| %>
-        <li class="message__day-heading"><%= date_heading(datetime) %></li>
-        <% records.each_with_index do |note, note_index| %>
-          <% if note_index == (records.size - 1) && day_index == (@all_notes_by_day.size - 1) %>
-            <%= render note.class.name == "Note" ? "note" : "system_note", note: note, is_last_item: true %>
-          <% else %>
-            <%= render note.class.name == "Note" ? "note" : "system_note", note: note %>
+  <div class="slab slab--padded">
+    <div class="client-container">
+      <ul class="day-list">
+        <% if @all_notes_by_day.present? %>
+          <% @all_notes_by_day.each_with_index do |(datetime, records), day_index| %>
+            <li class="day-heading"><%= date_heading(datetime) %></li>
+            <% records.each_with_index do |note, note_index| %>
+              <%= render "note", note: note, is_last_item: note_index == (records.size - 1) && day_index == (@all_notes_by_day.size - 1) %>
+            <% end %>
           <% end %>
-        <% end %>
+         <% end %>
+      </ul>
+    <hr/>
+    <div class="single-input-form">
```

Comment:
I love that you gave these forms a named class!
We might want to rename these to "inline form" or something, only because we have a lot of things on the client side that could be construed as "single input forms".